docs/farming.md

```diff
@@ -19,17 +20,46 @@ You can find these executables in the [Releases](https://github.com/subspace/sub
 
 > This is a ***non-incentivized*** testnet. Meaning there are no rewards in place at this time, and has absolutely no financial benefit to being run at this time.
 
-### A. To Farm With The Network (Online)
+<details><summary> 🖼️ Windows Instructions (Click to Expand)</summary>
+<p>
 
-> NOTE: replace these: `subspace-node-x86_64-*-snapshot`, `subspace-farmer-x86_64-*-snapshot`
-> in the below commands with the name of the file you downloaded for your operating system.
+### 📝 Windows Installation
 
-**Mac/Linux**
+1. Download the executables for your operating system from the [Releases](https://github.com/subspace/subspace/releases) tab.
+2. open a terminal, on Windows this is `Command Prompt` or `Powershell`
+3. In the terminal we will change to the Downloads directory using this command `cd Downloads`
+4. We will then start the node using the following command
+>(replace `INSERT_YOUR_ID` with a nickname you choose)
```

Comment:
This is usually typed as a comment in the shell command snippet starting  with `#`.

Also here and in other places let's start actual content in quotes with a whitespace to separate it from `>` character, just like it was before.